steve/util.py

```diff
@@ -378,10 +378,14 @@ def out(*output, **kw):
     sys.stdout.write(output + '\n')
 
 
-def load_json_files(config):
+def load_json_files(config, path=None):
```

Comment:
Mmm... I think this solution is overly complex. Generally any time you have a tri-state argument that takes different types, it's a bad API smell.

Beyond that, I think I need to tinker with it before I understand enough of what you're hitting to have other ideas.